src/ispd/gui/LoadConfigurationDialog.java

```diff
@@ -1148,20 +1153,22 @@ private void onCancelClick(final ActionEvent evt) {
         this.setVisible(false);
     }
 
-    private static GerarCarga loadGeneratorFromTableRow(final List row) {
-        return new CargaForNode(
+    private static WorkloadGenerator loadGeneratorFromTableRow(final List row
+            , final Supplier<Integer> idSupplier) {
+        return new PerNodeWorkloadGenerator(
```

Comment:
Should validate to make sure it's correct: the order here can easily be wrong unnoticeably.